flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/EnvironmentSettings.java

```diff
@@ -225,9 +225,9 @@ public String getExecutor() {
 
     @Internal
     public Map<String, String> toPlannerProperties() {
```

Comment:
We don't need the `toPlannerProperties` anymore. The refactoring can be similar to FLINK-23482.